internal/commands/inspect_builder.go

```diff
@@ -1,430 +1,83 @@
 package commands
 
 import (
-	"bytes"
-	"fmt"
-	"io"
-	"strings"
-	"text/tabwriter"
-	"text/template"
-
-	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
 	"github.com/buildpacks/pack"
-	"github.com/buildpacks/pack/internal/builder"
+	"github.com/buildpacks/pack/builder"
 	"github.com/buildpacks/pack/internal/config"
-	"github.com/buildpacks/pack/internal/dist"
-	"github.com/buildpacks/pack/internal/style"
 	"github.com/buildpacks/pack/logging"
 )
 
-const (
-	writerMinWidth     = 0
-	writerTabWidth     = 0
-	buildpacksTabWidth = 8
-	defaultTabWidth    = 4
-	writerPadChar      = ' '
-	writerFlags        = 0
-	none               = "(none)"
-)
+type BuilderWriter interface {
+	Print(
+		logger logging.Logger,
+		localRunImages []config.RunImage,
```

Comment:
I believe these are run images defined in the system's config, rather than either the local or remote builder. It feels distinct from just `runImages` which would be the superset of run images from all possible sources. In this context there is no correlated concept of a `remote` run image. I think a different term could be helpful to avoid overloading the term `local` in this context, but it would be useful to continue to reference these as a subset of run images, and imply their source as being local config.